libs/core/src/lib/utils/services/resize-observer.service.ts

```diff
@@ -0,0 +1,84 @@
+import { ElementRef, Injectable, OnDestroy } from '@angular/core';
```

Comment:
is that service replacement for the `resize-observable.ts`? then we should mark that approach as deprecated. 

sorry, but what reason to have this service? i mean what not okay with previous approach?